ddtrace/contrib/dbapi_async/__init__.py

```diff
@@ -61,7 +79,7 @@ async def _trace_method(self, method, name, resource, extra_tags, dbm_propagator
                     SqlInjection.report(evidence_value=args[0])
 
             # set analytics sample rate if enabled but only for non-FetchTracedCursor
-            if not isinstance(self, FetchTracedCursor):
+            if not isinstance(self, FetchTracedAsyncCursor):
```

Comment:
Is this a different bug? Is this issue documented the PR description

Reply:
Yes, I will update the description, tests were failing for this async class since this line was previously incorrect